src/core/a-animation.js

```diff
@@ -326,7 +326,7 @@ module.exports.AAnimation = registerElement('a-animation', {
      * Works the same as component mixins but reimplemented because animations
      * aren't components.
      */
-    applyMixin: {
+    handleMixinUpdate: {
```

Comment:
apply mixin seems a better name than handleMixinUpdate. When invoked from the `attachedCallback` method there's no mixin update anywhere.